core/src/test/java/org/axonframework/testutils/XStreamSerializerFactory.java

```diff
@@ -0,0 +1,22 @@
+package org.axonframework.testutils;
+
+import com.thoughtworks.xstream.XStream;
+import com.thoughtworks.xstream.security.ExplicitTypePermission;
+import org.axonframework.serializer.xml.XStreamSerializer;
+
+public class XStreamSerializerFactory {
```

Comment:
Make sure this one is mentioned in the readme file since CIM is probably just instantiating `XStreamSerializer` in the old, java17-incompatible way.